yt/testing.py

```diff
@@ -91,7 +92,7 @@ def assert_rel_equal(a1, a2, decimals, err_msg="", verbose=True):
 
 # tested: volume integral is 1.
 def cubicspline_python(
-    x: float | np.ndarray,
+    x: float | NDArray[np.floating],
```

Comment:
I guess the return annotation should be changed as well

Reply:
oh ya, thought I had put that in. maybe i forgot to commit it.

Reply:
Did you forget again ? 😅

Reply:
hah, sorry, didn't push any changes yet, only fixed the stray .cpp